src/Reactor.jl

```diff
@@ -247,6 +247,25 @@ end
     end
 end
 
+@inline function _calc_jacobiany_k_deriv!(jac::Array{Float64,2},xind::Int64,dkdx::Float64,reactantinds::Array{Int64,1},productinds::Array{Int64,1},cs::Array{Float64,1},V::Float64)
```

Comment:
Thanks for the comment! I will add a docstring! So I'm using this function for both V and ni because they have the same deriv except for the dkdV or dkdni part. x can be either V or ni.